solr/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN git clone https://github.com/aws/efs-utils && \
 # Install hostname resolution dependencies
 RUN apt-get install -y dnsutils
 
-# Fix Issue https://github.com/GSA/datagov-deploy/issues/3285
+# Fix Issue https://github.com/GSA/data.gov/issues/3285
```

Comment:
are we still using angularjs? if not, can we remove this?

Reply:
It's used in SOLR UI, but I think this can be removed.